llvm/lib/Transforms/Vectorize/VPlanTransforms.cpp

```diff
@@ -1764,6 +1764,10 @@ static VPRecipeBase *createEVLRecipe(VPValue *HeaderMask,
   using namespace llvm::VPlanPatternMatch;
   auto GetNewMask = [&](VPValue *OrigMask) -> VPValue * {
     assert(OrigMask && "Unmasked recipe when folding tail");
+    using namespace llvm::VPlanPatternMatch;
```

Comment:
Is this needed? Does the `using` at line 1764 not apply?

Reply:
Oops yes, removed